lib/bundle.js

```diff
@@ -103,31 +199,33 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     return systemBuilder.trace(firstModule, cfg);
```

Comment:
@guybedford This is actually what I wanted to do at first. This is simple and simple is elegant :) .  Somehow got deviated by the `normalization` step required in the `expandGlob` function initially.
   
Steps:

1. Find each glob expression in the `operations` array
2. Execute the glob. get all the module names.
3. Push the every module name in the place of the corresponding `glob exprasion` in the `operations` array with the original operator. 
4. Repeat.

This gives us the opportunity to support all the glob patterns not just `lib/**/* and lib/*`. The only problem is how the program would separate the glob pattern from the `normal module name`. Simple change in the user input can make this happen. Consider the bundle expression bellow:

  `module/a + [lib/**/*] - [lib/*] + module/b + [*(a|b|c)]`

where everything inside every `[..]` is the glob expression. This way we can support complex glob pattern like the last one and we don't have to go through any additional `normalization` steps either. Let me know your thoughts.